docs/topics/modules/approving_your_application.adoc

```diff
@@ -0,0 +1,14 @@
+[id="approving_your_application"]
+= Approving your application
+
+If the quickstart application in the _Stage_ environment runs as expected, approve your application and promote it to the _Run_ environment:
+
+. Return to the {osio} tab which displays the *Pipeline* view and click btn:[Input Required] at the *Approve* stage of the pipeline.
+
+. Click btn:[Promote] to promote the build from the _Stage_ environment to the _Run_ environment. The roll out process from _Stage_ to _Run_ requires a few minutes.
```

Comment:
I'd avoid using "roll out process". First, the language seems a bit informal to use, but also the UI still calls it "rollout" rather than roll out.

Reply:
Removed.